docroot/sites/all/themes/custom/wsdot_classic/templates/panels-pane--block--block-11.tpl.php

```diff
@@ -27,7 +27,7 @@
 
   <?php print render($title_prefix); ?>
   <?php if ($title): ?>
-    <h4 <?php print $title_attributes; ?> style="color:#fff; background: #710101;">
+    <h4 <?php print $title_attributes; ?> style="color:#fff; background: #710101;border-radius: 0;">
```

Comment:
I'm not seeing what the `border-radius: 0` changes? Was it affecting a specific browser?

Reply:
The red box container div.contextual-links-region.panel-pane pane-block.pane-block-11 has the following style: border: 1px solid #710101. It also has a defauly border radius of 0px. The red box heading h4.pane-title has a border radius of 2px, background color: #710101. This causes the white background of the container to show at the upper left and right corners of the heading in all browsers. It's barely perceptible, but maybe worth fixing.

Reply:
The red box container div.contextual-links-region.panel-pane.pane-block.pane-block-11 has the following style: border: 1px solid #710101. It also has a default border radius of 0px. The red box heading h4.pane-title has a border radius of 2px, background color: #710101. This causes the white background of the container to show at the upper left and right corners of the heading in all browsers. It's barely perceptible, but maybe worth fixing.

Reply:
Thanks. I see it now. Good catch.